### PR TITLE
Update TinyBERT modeling and CLUE results

### PR DESCRIPTION
**Pull request by LiuChiachi**
### PR types

New features
### PR changes

APIs
### Description

Update TinyBert, make it support multiple choice and question answering tasks.

训练验证后可合入

### CODE REVIEW

File: examples/benchmark/clue/README.md [hunk 1]
```diff
@@ -359,6 +359,81 @@
                           <span style="font-size:18px"><b>69.73</b></span>
                         </td>
                 </tr>
+                <tr>
+                <td style="text-align:center">
+                        <span style="font-size:18px">Tinybert<sub>6</sub>, Chinese</span>
```
Comment (tianxin1860) [inline examples/benchmark/clue/README.md]: Tinybert -> TinyBERT

File: examples/benchmark/clue/README.md [hunk 2]
```diff
@@ -424,20 +499,22 @@ AFQMC、TNEWS、IFLYTEK、CMNLI、OCNLI、CLUEWSC2020、CSL 、CHID 和 C<sup>3<
...
+| ERNIE 3.0-Base-zh             | 3e-5,16 | 3e-5,32 | 5e-5,32 | 3e-5,32  | 2e-5,64  | 2e-5,16     | 2e-5,32 | 2e-5,24  | 3e-5,24 | 3e-5,32       |
+| ERNIE-Gram-zh                 | 1e-5,16 | 5e-5,16 | 5e-5,16 | 2e-5,32  | 2e-5,64  | 3e-5,16     | 3e-5,64 | 3e-5,32  | 2e-5,24 | 2e-5,24       |
+| Mengzi-Bert-Base              | 3e-5,32 | 5e-5,32 | 5e-5,16 | 2e-5,16  | 2e-5,16  | 3e-5,8      | 1e-5,16 | 3e-5,24  | 3e-5,24 | 2e-5,32       |
+| ERNIE-1.0                     | 3e-5,16 | 3e-5,32 | 5e-5,16 | 5e-5，32 | 3e-5，16 | 2e-5,8      | 2e-5,16 | 3e-5,32  | 3e-5,24 | 3e-5,24       |
+| RoBERTa-wwm-ext               | 3e-5,32 | 3e-5,64 | 5e-5,16 | 3e-5,32  | 2e-5,32  | 3e-5,32     | 2e-5,32 | 3e-5,32  | 2e-5,32 | 3e-5,24       |
+| Bert-Base-Chinese             | 2e-5,16 | 5e-5,16 | 5e-5,16 | 5e-5,64  | 3e-5,16  | 3e-5,16     | 1e-5,16 | 3e-5,24  | 2e-5,32 | 3e-5,24       |
+| ERNIE 3.0-Medium-zh           | 3e-5,32 | 3e-5,64 | 5e-5,32 | 2e-5,32  | 1e-5,64  | 3e-5,16     | 2e-5,32 | 3e-5,24  | 2e-5,24 | 1e-5,24       |
+| Tinybert<sub>6</sub> ,Chinese | 1e-5,16 | 3e-5,32 | 5e-5,16 | 5e-5,32  | 3e-5,64  | 3e-5,16     | 3e-5,16 | 8,3e-5   | 3e-5,24 | 2e-5,24       |
```
Comment (tianxin1860) [inline examples/benchmark/clue/README.md]: 同上

File: model_zoo/ernie-3.0/README.md [hunk 3]
```diff
@@ -359,6 +359,82 @@
                           <span style="font-size:18px"><b>69.73</b></span>
                         </td>
                 </tr>
+                <tr>
+                <td style="text-align:center">
+                        <span style="font-size:18px">Tinybert<sub>6</sub>, Chinese</span>
```
Comment (tianxin1860) [inline model_zoo/ernie-3.0/README.md]: 同上

File: examples/benchmark/clue/README.md [hunk 4]
```diff
@@ -424,20 +499,22 @@ AFQMC、TNEWS、IFLYTEK、CMNLI、OCNLI、CLUEWSC2020、CSL 、CHID 和 C<sup>3<
...
+| Model                         | AFQMC   | TNEWS   | IFLYTEK | CMNLI    | OCNLI    | CLUEWSC2020 | CSL     | CMRC2018 | CHID    | C<sup>3</sup> |
+| ----------------------------- | ------- | ------- | ------- | -------- | -------- | ----------- | ------- | -------- | ------- | ------------- |
+| RoBERTa-wwm-ext-large         | 1e-5,32 | 3e-5,32 | 2e-5,32 | 1e-5,16  | 1e-5,16  | 2e-5,16     | 2e-5,16 | 3e-5,32  | 1e-5,24 | 2e-5,24       |
+| ERNIE 3.0-Base-zh             | 3e-5,16 | 3e-5,32 | 5e-5,32 | 3e-5,32  | 2e-5,64  | 2e-5,16     | 2e-5,32 | 2e-5,24  | 3e-5,24 | 3e-5,32       |
+| ERNIE-Gram-zh                 | 1e-5,16 | 5e-5,16 | 5e-5,16 | 2e-5,32  | 2e-5,64  | 3e-5,16     | 3e-5,64 | 3e-5,32  | 2e-5,24 | 2e-5,24       |
+| Mengzi-Bert-Base              | 3e-5,32 | 5e-5,32 | 5e-5,16 | 2e-5,16  | 2e-5,16  | 3e-5,8      | 1e-5,16 | 3e-5,24  | 3e-5,24 | 2e-5,32       |
+| ERNIE-1.0                     | 3e-5,16 | 3e-5,32 | 5e-5,16 | 5e-5，32 | 3e-5，16 | 2e-5,8      | 2e-5,16 | 3e-5,32  | 3e-5,24 | 3e-5,24       |
```
Comment (ZeyuChen) [inline examples/benchmark/clue/README.md]: ERNIE 1.0 没有横杆

**Review by tianxin1860 — COMMENTED** [review]
Leave a comment

File: examples/benchmark/clue/README.md [hunk 5]
```diff
@@ -424,20 +499,22 @@ AFQMC、TNEWS、IFLYTEK、CMNLI、OCNLI、CLUEWSC2020、CSL 、CHID 和 C<sup>3<
...
+| RoBERTa-wwm-ext               | 3e-5,32 | 3e-5,64 | 5e-5,16 | 3e-5,32  | 2e-5,32  | 3e-5,32     | 2e-5,32 | 3e-5,32  | 2e-5,32 | 3e-5,24       |
+| Bert-Base-Chinese             | 2e-5,16 | 5e-5,16 | 5e-5,16 | 5e-5,64  | 3e-5,16  | 3e-5,16     | 1e-5,16 | 3e-5,24  | 2e-5,32 | 3e-5,24       |
```
Comment (ZeyuChen) [inline examples/benchmark/clue/README.md]: BERT

Comment (ZeyuChen) [reply examples/benchmark/clue/README.md]: 模型名需要准确严谨

File: examples/benchmark/clue/README.md (hunk 2 above)
Comment (ZeyuChen) [inline examples/benchmark/clue/README.md]: TinyBERT
BERT要大写

File: examples/benchmark/clue/README.md [hunk 6]
```diff
@@ -424,20 +499,22 @@ AFQMC、TNEWS、IFLYTEK、CMNLI、OCNLI、CLUEWSC2020、CSL 、CHID 和 C<sup>3<
...
+| ERNIE 3.0-Medium-zh           | 3e-5,32 | 3e-5,64 | 5e-5,32 | 2e-5,32  | 1e-5,64  | 3e-5,16     | 2e-5,32 | 3e-5,24  | 2e-5,24 | 1e-5,24       |
+| Tinybert<sub>6</sub> ,Chinese | 1e-5,16 | 3e-5,32 | 5e-5,16 | 5e-5,32  | 3e-5,64  | 3e-5,16     | 3e-5,16 | 8,3e-5   | 3e-5,24 | 2e-5,24       |
+| Roformer-v2-small             | 5e-5,16 | 2e-5,16 | 5e-5,16 | 5e-5,32  | 2e-5,16  | 3e-5,8      | 3e-5,16 | 3e-5,24  | 3e-5,24 | 3e-5,24       |
```
Comment (ZeyuChen) [inline examples/benchmark/clue/README.md]: RoFormer

**Review by ZeyuChen — COMMENTED** [review]
把所有模型名整正确吧

**Review by ZeyuChen — APPROVED** [review]

**Pull request merged**